x/cronos/keeper/precompiles/relayer.go

```diff
@@ -81,7 +80,7 @@ func (bc *RelayerContract) Run(evm *vm.EVM, contract *vm.Contract, readonly bool
 	}
 	prefix := int(binary.LittleEndian.Uint32(contract.Input[:prefixSize4Bytes]))
 	input := contract.Input[prefixSize4Bytes:]
-	stateDB := evm.StateDB.(precompiles.ExtStateDB)
+	stateDB := evm.StateDB.(ExtStateDB)
```

Comment:
```suggestion
	stateDB := ExtStateDB(evm.StateDB)
```

turn it to compile time check

Reply:
seems cant do in compile time, which dont know concrete type, unless we change evm.StateDB type to ExtStateDB

Reply:
I see, it's gone through go-ethereum